### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**
Initial PR

### CODE REVIEW

File: mayhem/fuzz_pcode.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env python3
```
Comment (xansec) [inline mayhem/fuzz_pcode.py:1]: 0

**Review by xansec — COMMENTED** [review]
Looking to hear back on non-deterministic exception handling.

**Comment by capuanob** [comment]
@xansec Fixed thanks to your help!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**